test/e2e/agent/recipes_test.go

```diff
@@ -133,6 +133,9 @@ func TestFleetKubernetesIntegrationRecipe(t *testing.T) {
 }
 
 func TestFleetCustomLogsIntegrationRecipe(t *testing.T) {
+
+	test.SkipUntilResolution(t, 6331)
```

Comment:
Can all of the skip lines just use this?

Reply:
_(Damn i'm spotted. I wrote this and then I thought stop you are going too far and ... I managed to leave a reference :sweat_smile:.)_

Yes :)